libraries/protocol/include/steemit/protocol/config.hpp

```diff
@@ -43,6 +42,8 @@
 #define STEEMIT_OWNER_AUTH_HISTORY_TRACKING_START_BLOCK_NUM 1
 #else // IS LIVE STEEM NETWORK
 
+#define STEEMIT_BLOCKCHAIN_VERSION              ( version(0, 19, 0) )
```

Comment:
Changes in develop mean that we do not need to have a testnet version and mainnet version that are different.